cpp/EncryptionProvider.cpp

```diff
@@ -0,0 +1,15 @@
+//
+// Created by hyu on 2/5/18.
+//
+
+#include "EncryptionProvider.hpp"
+
+char* Snowflake::Client::EncryptionProvider::encrypt()
+{
+  return "";
```

Comment:
This is causing compilation error.

Actually it is warning but we want to fix all warnings for safetly.